packages/javascript/bh-shared-ui/src/components/ExploreTable/explore-table-utils.ts

```diff
@@ -11,20 +12,32 @@ export const makeStoreMapFromColumnOptions = (columnOptions: ManageColumnsComboB
         {} as Record<string, boolean>
     );
 
+const KNOWN_NODE_KEYS = ['kind', 'objectId', 'label', 'isTierZero'];
+
+export const getExploreTableData = (graphData: GraphResponse | FlatGraphResponse | undefined) => {
+    if (!graphData || !isGraphResponse(graphData)) return;
+
+    const nodes = graphData.data.nodes;
+    const unknownNodeKeys = graphData.data.node_keys;
+
+    const completeNodeKeys = unknownNodeKeys?.concat(KNOWN_NODE_KEYS);
+
+    return {
+        nodes,
+        node_keys: completeNodeKeys,
+    };
+};
+
 export type NodeClickInfo = { id: string; x: number; y: number };
 export type MungedTableRowWithId = GraphNodeSpreadWithProperties & { id: string };
 
-const REQUIRED_EXPLORE_TABLE_COLUMN_KEYS = ['nodetype', 'objectid', 'name', 'isTierZero'];
-
-export const requiredColumns = Object.fromEntries(REQUIRED_EXPLORE_TABLE_COLUMN_KEYS.map((key) => [key, true]));
+export const requiredColumns = Object.fromEntries(KNOWN_NODE_KEYS.map((key) => [key, true]));
```

Comment:
I changed some of these fields to match the fields that are above the properties bag in the `GET /cypher` response body because although objectid, nodetype, and name map to the objectId, kind, and label fields we dont have as much control in the property bag and those could change without us knowing!

I think I figured out the fallout from that change, but youd probably know any other places that would need to be updated too!